seatunnel-api/src/main/java/org/apache/seatunnel/api/serialization/DeserializationSchema.java

```diff
@@ -34,12 +35,30 @@ public interface DeserializationSchema<T> extends Serializable {
      */
     T deserialize(byte[] message) throws IOException;
 
+    /**
+     * Deserializes the byte message.
+     *
+     * @param message The message, as a byte array.
+     * @param tablePath The Registry name.
+     * @return The deserialized message as an SeaTunnel Row (null if the message cannot be
+     *     deserialized).
+     */
+    T deserialize(byte[] message, TablePath tablePath) throws IOException;
```

Comment:
Can’t we distinguish table formats through record topic?

Reply:
> Can’t we distinguish table formats through record topic?

Currently pr supports that each topic can specify a different format
![image](https://github.com/apache/seatunnel/assets/76689593/986221d7-bb44-4c9d-a071-d17ee83db724)

Reply:
> 1. A topic should not have multiple data formats
> 2. Each topic should correspond to a fixed data format
The current logic should only have one data format per topic, if it is a regular matching topic will also force a format, unless it can be different topics we can let the user set different formats